src/main.py

```diff
@@ -199,7 +199,7 @@ def main():
     new_custom_layers = Dense(1024, activation="relu")(new_custom_layers)
     new_custom_layers = Dropout(0.5)(new_custom_layers)
     new_custom_layers = Dense(1024, activation="relu")(new_custom_layers)
-    predictions = Dense(train_generator.num_class, activation="softmax")(new_custom_layers)
+    predictions = Dense(train_generator.num_classes, activation="softmax")(new_custom_layers)
```

Comment:
I think it looks better this way (for backward compatibility):
```suggestion
try:
    predictions = Dense(self.train_generator.num_classes,
                        activation="softmax")(new_custom_layers)
except AttributeError:
    predictions = Dense(self.train_generator.num_class,
                        activation="softmax")(new_custom_layers)
```